### PR TITLE
wizma

### PR DESCRIPTION
**Pull request by wizma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # simple-java-maven-app
-wizma
+wizma 1
 A simple Java application using Maven for use with the CloudBees CI Workshop.
```
Comment (peterli-rogers) [inline README.md:4]: need to add "wizma" here and there

Comment (peterli-rogers) [inline README.md:4]: done ?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 
 The CloudBees CI Pipeline Template used with this repository requires a specific file in each branch in order for that branch to be indexed and processed by the template job. That file name must be `cloudbees-ci.yml`. An example file, name `example.cloudbees-ci.yml` has been provided in this repository and all you have to do is rename that file to `cloudbees-ci.yml` to have the branch processed by the CloudBees CI template.
```
Comment (peterli-rogers) [inline README.md:5]: add "wizma" here as well

**Review by peterli-rogers — COMMENTED** [review]
please take a look

File: README.md (hunk 1 above)
Comment (wizma) [reply README.md:4]: done

**Review by peterli-rogers — CHANGES_REQUESTED** [review]
request changes

**Review by peterli-rogers — APPROVED** [review]

**Pull request merged**